.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
Fails here with: 

``
PHP Fatal error:  Class 'Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase' not found in /home/travis/build/symfony-cmf/SeoBundle/Tests/Unit/DependencyInjection/Compiler/RegisterExtractorsPassTest.php on line 21
```